build-tools/packages/build-cli/src/library/repoPolicyCheck/assertShortCode.ts

```diff
@@ -123,7 +123,9 @@ export const handler: Handler = {
 								);
 							};
 
+							// eslint-disable-next-line max-depth
 							if (shouldRemoveSurroundingQuotes(originalErrorText)) {
+								// eslint-disable-next-line unicorn/prefer-string-slice
```

Comment:
Don't like string slice?

Reply:
Just don't want to churn the code if it is working.